<launch>
  
  <node pkg="ti_mmwave_rospkg" type="ti_data_processor" name="ti_data_processor_node" output="screen" />

  <!-- Static transform from map to base_radar_link for visualization of stand-alone mmWave sensor using Rviz -->
  <node pkg="tf" type="static_transform_publisher" name="static_tf_map_to_base_radar_link" args="0 0 0 0 0 0 ti_mmwave_pcl ti_mmwave 30"/>

  <!-- ti_data_processor node-->
  <!--<node pkg="ti_mmwave_rospkg" type="ti_data_processor.py" name="ti_data_processor_node" output="screen" />-->

  <!-- Launch Rviz with pre-defined configuration to view mmWave sensor detected object data (color by intensity) -->
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find ti_mmwave_rospkg)/launch/rviz/ti_mmwave_3d.rviz"/>

</launch>
